helpers.js

```diff
@@ -35,6 +35,7 @@ function scrobbleAnime(animeId, episode) {
             };
         fetch(url, options);
         $('#anilist_scrobbler_notice').text(chrome.i18n.getMessage("appName") + ' : ' + chrome.i18n.getMessage("scrobbling_ok"));
+        clearInterval();
```

Comment:
`clearInterval()` needs an argument, the setInterval var, so you must define a varname to the setInterval or create a destroy function in Timer class

Reply:
Fixed in the next commit!